tests/test_utils.py

```diff
@@ -28,6 +28,7 @@
 from art.utils import second_most_likely_class, random_targets, get_label_conf, get_labels_np_array, preprocess
 from art.utils import segment_by_class, performance_diff
 from art.utils import is_probability
+from art.utils import check_pytorch_loss_param
```

Comment:
Is this import still required with the currently implemented solution of listing all loss functions?

Reply:
Removed.